src/app/technical-record-create/technical-record-create.component.scss

```diff
@@ -1,22 +1,4 @@
-@import "node_modules/govuk-frontend/govuk/all";
-
-.create-tech-record {
-  & h1:first-child {
-    margin-bottom: 1rem;
-  }
-  & .govuk-grid-row {
-    margin-bottom: 1.5rem;
-  }
-
-  & a#test-continue-btn {
-    padding: 10px 15px 10px 15px;
-    font-size: 20px;
-  }
-
-  & .govuk-error-summary {
-    padding-bottom: 0 !important;
-    & .govuk-error-message {
-      text-decoration: underline;
-    }
-  }
+.govuk-back-link {
+  text-decoration: underline;
```

Comment:
we will have to check on that one as i suspect we are modifying the style of the back button to match some design or based on feedback however by doing so we are not being compliant with GDS design system.
I am happy to pick that up separately in fact if we need @irina005 as I believe we should be using GDS living styleguide out of the box and if changes are required then the team should have had a request submitted to check whether they agree on it.